src/olympia/abuse/utils.py

```diff
@@ -70,13 +76,18 @@ def notify_owners(self, *, policy_text=None):
         owners = self.get_owners()
         if not owners:
             return
-        name = self.get_target_name()
+        with no_jinja_autoescape():
```

Comment:
Could this be solved by escaping the quotes in the linked issue? Why disable all escaping?

Reply:
It wasn't just the quotes, and potentially could happen to other things than just the name / ratings.